### PR TITLE
xfce-extra/xfce4-cpugraph-plugin: panel-4.15 compatibility

### PR DESCRIPTION
**Pull request by atomsymbol**
Please merge. Thanks.

Closes: https://bugs.gentoo.org/730886
Closes: https://bugs.gentoo.org/744058

### CODE REVIEW

File: xfce-extra/xfce4-cpugraph-plugin/files/1.1.0-panel-4.15.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -Naur xfce4-cpugraph-plugin-1.1.0/panel-plugin/cpu.h xfce4-cpugraph-plugin-1.1.0-patched/panel-plugin/cpu.h
```
Comment (mgorny) [inline xfce-extra/xfce4-cpugraph-plugin/files/1.1.0-panel-4.15.patch:1]: Please use a git patch with correct metadata. You can grab it from: https://gitlab.xfce.org/panel-plugins/xfce4-cpufreq-plugin/-/commit/30f6de22897798cb1d8b6a18d506e9e331a97b10.patch

Also start the filename with package name, i.e. xfce4-....patch

**Review by mgorny — CHANGES_REQUESTED** [review]
Also I don't see the fix for #744058.

File: xfce-extra/xfce4-cpugraph-plugin/files/1.1.0-panel-4.15.patch (hunk 1 above)
Comment (atomsymbol) [reply xfce-extra/xfce4-cpugraph-plugin/files/1.1.0-panel-4.15.patch]: Just a note: This pull request is for cpugraph plugin, not cpufreq plugin.

Comment (mgorny) [reply xfce-extra/xfce4-cpugraph-plugin/files/1.1.0-panel-4.15.patch]: Yeah, sorry, gave you the wrong link.

**Comment by atomsymbol** [comment]
> Also I don't see the fix for #744058.

The change in the include directive also happens to fix #744058. Tested with clang 11.0.0.

**Comment by mgorny** [comment]
Ok, thanks.

**Review by mgorny — APPROVED** [review]

**Pull request closed without merging**